apps/backend/src/app/categories/dto/create-category.dto.ts

```diff
@@ -0,0 +1,6 @@
+import { IsString } from 'class-validator';
+
+export class CreateCategoryDto {
+  @IsString()
+  readonly name: string;
```

Comment:
Any minimum/maximum length?

Reply:
OK, updated, please check if it's ok now :)